model/clusters_mgmt/v1/wif_config_type.model

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Definition of an wif_config resource.
+class WifConfig {
+	// The name OCM clients will display for this wif_config.
+	DisplayName String
+	// Holds GCP related data.
+	Gcp WifGcp
+	// The OCM organization that this wif_config resource belongs to.
+	Organization OrganizationLink
+	// Holds time based data related to the wif_config resource.
+	Time WifTime
+}
+
+struct WifGcp {
+	// This is the service account email that OCM will use to access other SAs. 
+	ImpersonatorEmail String
+	// This represents the GCP project ID in which the wif resources will be configured.
+	ProjectId String
+	// The list of service accounts and their associated roles that will need to be
+	// configured on the user's GCP project.
+	ServiceAccounts []WifServiceAccount
+	// The workload identity configuration data that will be used to create the
+	// workload identity pool on the user's account.
+	WorkloadIdentityPool WifWorkloadIdentityPool
+}
+
+struct WifWorkloadIdentityPool {
+	// Identity provider configuration data that will be created as part of the
+	// workload identity pool.
+	IdentityProvider WifIdentityProvider
+	// The Id of the workload identity pool.
+	PoolId String
+	// The display name of the workload identity pool.
+	PoolName String
+}
+
+struct WifIdentityProvider {
+	AllowedAudiences []String
+	IdentityProviderId String
+	IssuerUrl String
+	Jwks string
+}
+
+struct WifTime {
+	// Date and time when the wif_config was initially created.
+	CreatedAt Date
+	// Date and time when the wif_config was last checked.
+	LastChecked Date
```

Comment:
Time is mainly to determine if a WifConfig is stale \ hanging around too long and needs to be cleanedup.
Given that LastUsed makes more sense than lastChecked.

Reply:
As discussed during office hours, this structure and its associated field within the WifConfig resource has been removed.